### ISSUE TITLE
Anyone know why this warning show up?

### ISSUE THREAD

**Issue by jianjianGJ** [rusty1s/pytorch_geometric#1287]
## ❓ Questions & Help
My environment is **ubuntu16.04+torch1.5+cu101+pytorch-geometric** .
when I run the example such as **sgc.py,gat.py** in examples ,it's training ok, but a lot of warnings show up:
`Warning: indexing with dtype torch.uint8 is now deprecated, please use a dtype torch.bool instead. (expandTensors at /pytorch/aten/src/ATen/native/IndexingUtils.h:20)`
I add code `mask = mask.bool() ` before mask was used, but the warnings  still  there.I know there is no such thing in the elder version.
Anyone can help me? Thanks so much!!

**Comment by jianjianGJ** [comment]
use `data.train_mask.bool()` instead of `data.train_mask`

**Comment by rusty1s** [comment]
That should have been fixed a long time ago. Which version of PyG are you using?

**Comment by jianjianGJ** [comment]
Yes,you are right, that's because I use the processed data which was generated a long time ago. I delete the old processed data and reprocess the raw data ,the warnings never show again.
By the way.Thank you for your contribution to the community!!!

**Comment by rusty1s** [comment]
Perfect :) Thank you!
